tree/dataframe/inc/ROOT/RDataSource.hxx

```diff
@@ -190,31 +191,46 @@ public:
    /// \brief Convenience method called before starting an event-loop.
    /// This method might be called multiple times over the lifetime of a RDataSource, since
    /// users can run multiple event-loops with the same RDataFrame.
-   /// Ideally, `Initialise` should set the state of the RDataSource so that multiple identical event-loops
+   /// Ideally, `Initialize` should set the state of the RDataSource so that multiple identical event-loops
    /// will produce identical results.
    // clang-format on
-   virtual void Initialise() {}
+   virtual void Initialize() {}
+
+   /// \cond
+   R__DEPRECATED(6,30,"Use Initialize (with a z) instead.") void Initialise() { Initialize(); }
```

Comment:
Doesn't this need to be `virtual` to not break expert users' derived classes? And even then... someone needs to call it?